NCsvPerf/CsvReadable/Implementations/FlatFiles.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Knapcode.NCsvPerf.CsvReadable
+{
+    /// <summary>
+    /// Package: https://www.nuget.org/packages/FlatFiles
+    /// Source: https://github.com/jehugaleahsa/FlatFiles
+    /// </summary>
+    public class FlatFiles : ICsvReader
+    {
+        private readonly ActivationMethod _activationMethod;
+
+        public FlatFiles(ActivationMethod activationMethod)
+        {
+            _activationMethod = activationMethod;
+        }
+
+        public List<T> GetRecords<T>(MemoryStream stream) where T : ICsvReadable, new()
+        {
+            var activate = ActivatorFactory.Create<T>(_activationMethod);
+            var allRecords = new List<T>();
+
+            using (var reader = new StreamReader(stream))
+            {
+                var csvReader = new global::FlatFiles.SeparatedValueReader(reader);
+
+                while (csvReader.Read())
+                {
+                    var values = csvReader.GetValues();
+                    var record = activate();
+                    record.Read(i => values[i]?.ToString() ?? string.Empty);
```

Comment:
Is the first `?` necessary? When would `values[i]` return null? Ideally we have minimal null coalescence since that is extra operations and will yield worse performance.

Reply:
It is null instead of an empty string when the field is empty. Maybe there's a setting to change that.

Reply:
I don't see anything in options. If you have a faster way of checking for a null and returning a string if it is, let me know. I have never performance tested that scenario.

Reply:
Okay, makes sense!